<launch>

    <node pkg="object_detect" name="object_detect_node" type="inference.py" output="screen">
        <param name="image_topic" value="/usb_cam/image_raw" />
    </node> 

</launch>
